packages/@react-spectrum/s2/src/Badge.tsx

```diff
@@ -73,42 +78,103 @@ const badge = style<BadgeStyleProps>({
-      orange: 'black',
-      yellow: 'black',
-      charteuse: 'black',
-      celery: 'black'
+    fillStyle: {
+      bold: {
+        default: 'white',
+        variant: {
+          notice: 'black',
+          orange: 'black',
+          yellow: 'black',
+          charteuse: 'black',
+          celery: 'black'
+        }
+      },
+      subtle: 'gray-1000',
+      outline: 'gray-1000'
     }
   },
   backgroundColor: {
-    variant: {
-      accent: 'accent',
-      informative: 'informative',
-      neutral: 'neutral-subdued',
-      positive: 'positive',
-      notice: 'notice',
-      negative: 'negative',
-      gray: 'gray',
-      red: 'red',
-      orange: 'orange',
-      yellow: 'yellow',
-      charteuse: 'chartreuse',
-      celery: 'celery',
-      green: 'green',
-      seafoam: 'seafoam',
-      cyan: 'cyan',
-      blue: 'blue',
-      indigo: 'indigo',
-      purple: 'purple',
-      fuchsia: 'fuchsia',
-      magenta: 'magenta',
-      pink: 'pink',
-      turquoise: 'turquoise',
-      brown: 'brown',
-      cinnamon: 'cinnamon',
-      silver: 'silver'
+    fillStyle: {
+      bold: {
+        variant: {
+          accent: 'accent',
+          informative: 'informative',
+          neutral: 'neutral-subdued',
+          positive: 'positive',
+          notice: 'notice',
+          negative: 'negative',
+          gray: 'gray',
+          red: 'red',
+          orange: 'orange',
+          yellow: 'yellow',
+          charteuse: 'chartreuse',
+          celery: 'celery',
+          green: 'green',
+          seafoam: 'seafoam',
+          cyan: 'cyan',
+          blue: 'blue',
+          indigo: 'indigo',
+          purple: 'purple',
+          fuchsia: 'fuchsia',
+          magenta: 'magenta',
+          pink: 'pink',
+          turquoise: 'turquoise',
+          brown: 'brown',
+          cinnamon: 'cinnamon',
+          silver: 'silver'
+        }
+      },
+    subtle: {
+      variant: {
+        accent: 'accent-200',
+        informative: 'informative-subtle',
+        neutral: 'neutral-subtle',
+        positive: 'positive-subtle',
+        notice: 'notice-subtle',
+        negative: 'negative-subtle',
+        gray: 'gray-100',
+        red: 'red-200',
+        orange: 'orange-200',
+        yellow: 'yellow-200',
+        charteuse: 'chartreuse-200',
+        celery: 'celery-200',
+        green: 'green-200',
+        seafoam: 'seafoam-200',
+        cyan: 'cyan-200',
+        blue: 'blue-200',
+        indigo: 'indigo-200',
+        purple: 'purple-200',
+        fuchsia: 'fuchsia-200',
+        magenta: 'magenta-200',
+        pink: 'pink-200',
+        turquoise: 'turquoise-200',
+        brown: 'brown-200',
+        cinnamon: 'cinnamon-200',
+        silver: 'silver-200'
+        }
+      },
+      outline: 'gray-25'
+    }
+  },
+  borderStyle: {
+    fillStyle: {
+      outline: 'solid'
+    }
+  },
+  boxSizing: 'border-box',
+  borderWidth: '[2px]',
+  borderColor: {
+    fillStyle: {
```

Comment:
add these
```
      subtleFill: 'transparent',
      boldFill: 'transparent'
```

this will also help with alignment across the ones with visible borders and the ones without, right now if you toggle between the two, the contents of a badge will move